backend/apps/student/models.py

```diff
@@ -79,6 +80,15 @@ def __str__(self):
     def get_absolute_url(self) -> str:
         return reverse('student:detail', args=[self.pk])
 
+    def can_pin(self) -> bool:
+        membership = apps.get_model('group.Membership')
+        return (membership.objects
+                .filter(student=self,
+                        admin=True,
+                        group__can_pin=True,
+                        group__archived=False)
+                .exists() or self.user.is_staff)

```

Comment:
```suggestion
                .exists() or self.user.is_superuser)
```

`is_staff` siginifie que l'utilisateur a accès à l'interface admin, mais il n'est pas forcément super admin